wiki/pl.json

```diff
@@ -0,0 +1,14 @@
+{
+	"achievement-page-title": "Osiągnięcia",
+	"achievement-page-description": "Lista osiągnięć Manufactur'inc na Steam.",
+	"contracts-page-title": "Kontrakty",
+	"contract-page-title": "Kontrakt",
+	"equipments-page-title": "Equipments", //Review and context needed. What do you mean by equipment? Machines? 
+	"equipment-page-title": "Equipment", //^^
+	"equipment-page-description": "Ta karta pokazuje informację na temat ekwipunku \"%element%\" w Manufactur'inc.", //Review and context needed
```

Comment:
This is basically a description for Google and the other search engines.
It's the wiki page for a specific equipment. E.g: https://wiki.dysnomia.studio/manufactur-inc/equipment/small_saw/

Reply:
Same problem as the one above. We need to settle on one word that could describe the idea of those things. I guess "infrastructure" would work in this context

Reply:
Nevermind. Let's just use what I settled on which is a word "wyposażenie". It works in this context